### PR TITLE
jesse/fix-nightly-v1.0.0-rc1

### PR DESCRIPTION
**Pull request by gatsbyz**

### CODE REVIEW

**Review by praetoriansentry — APPROVED** [review]

File: .github/workflows/deploy.nightly.devnet.yml [hunk 1]
```diff
@@ -188,6 +202,9 @@ jobs:
 
           main
 
+          sudo apt update
+          sudo apt install nodejs npm
```
Comment (Stefan-Ethernal) [inline .github/workflows/deploy.nightly.devnet.yml]: Why do we need npm and Node JS?

**Review by Stefan-Ethernal — COMMENTED** [review]
Can you provide a bit more context, what was wrong with the previous version of the nightly workflow?

File: .github/workflows/deploy.nightly.devnet.yml (hunk 1 above)
Comment (gatsbyz) [reply .github/workflows/deploy.nightly.devnet.yml]: i got rid of this

**Comment by gatsbyz** [comment]
> Can you provide a bit more context, what was wrong with the previous version of the nightly workflow?

@Stefan-Ethernal I updated the script to use the `--test` flag. The `stake` command was failing with the error `staking transaction failed on block`. And this is because a ERC20StakeToken needs to be deployed. We learned that the `--test` flag is doing this in the back, so we included it in this nightly script. Hope this explains.

**Review by vcastellm — APPROVED** [review]

**Pull request merged**